src/components/sidebar/MenuMap.js

```diff
@@ -25,6 +26,15 @@ class MenuMap extends Component {
     e.stopPropagation()
     this.props.selectSolution(e.currentTarget.id.slice(10))
   }
+  toggleMenu = (e) => {
+    e.stopPropagation()
+    if(e.currentTarget.open) {
+      this.props.toggleMenu({key: this.props.name, val: 'open'})
+    }
+    else {
+        this.props.toggleMenu({key: this.props.name, val: 'false'})
```

Comment:
maybe instead of `{key: this.props.name, val: 'false'}` we can do `{ name: this.props.name, open: true }`, I think this will make the code more readable and will prevent us from having to do comparisons like `if (val === 'open')`. Instead, we could just do `if (open)`

Reply:
Done